content/docs/reference-react-dom-server.md

```diff
@@ -6,40 +6,40 @@ category: Reference
 permalink: docs/react-dom-server.html
 ---
 
-The `ReactDOMServer` object enables you to render components to static markup. Typically, it's used on a Node server:
+O objeto `ReactDOMServer` permite que você renderize componentes para markup estático. Normalmente, é usado em um servidor Node:
 
 ```js
-// ES modules
+// Módulos ES
 import ReactDOMServer from 'react-dom/server';
 // CommonJS
 var ReactDOMServer = require('react-dom/server');
 ```
 
-## Overview {#overview}
+## Visão Geral {#overview}
```

Comment:
@fjoshuajr you mentioned in a previous PR that we should not translate links. could you elaborate? seems ok to me

Reply:
Yeah, I made a mistake those can be translated as well. My bad!